mqr/tests/test_models.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
```

Comment:
I'm not sure if these tests are needed, they seems to be testing the django `get_or_create` function, it's mostly safe to assume those are tested by them